spec/factories/resources.rb

```diff
@@ -13,7 +13,7 @@
   factory :resource do
     name { "MyString" }
     resource_type { "MyString" }
-    status { "MyString" }
-    room { nil }
+    # room_id { Faker::Number.number(digits: 7) }
```

Comment:
I don't think we need to keep room_id in the factory file. `association :room` is enough.
You can try to run this test
```
context "the Factory" do
    it 'is valid' do
      r = create(:resource)
      binding.pry
      expect(FactoryBot.create(:resource)).to be_valid
    end
  end
```
When it stops, type 'r' in the console. You will see that the resource object has room_id.
You can type `Room.find(r.room_id) `to see the room.
Type '!!!' to exit the console.